src/views/components/copy-dropdown.html

```diff
@@ -0,0 +1,33 @@
+<div class="c-copy-dropdown btn-group {{#if ~/dropup}}dropup{{/if}}">
+	<span data-toggle="dropdown">
+		<i class="fa fa-clipboard" aria-hidden="true"></i>
+		<i class="fa fa-angle-down" aria-hidden="true"></i>
+	</span>
+
+	<ul class="dropdown-menu dropdown-menu-right">
+		<li><a as-clipboard="undefined, undefined, '.c-copy-dropdown'" data-clipboard-text="{{url}}">Copy URL</a></li>
+		<li><a as-clipboard="undefined, undefined, '.c-copy-dropdown'" data-clipboard-text="{{hash}}">Copy SRI</a></li>
```

Comment:
This no longer contains the `sha256-` prefix. Maybe this is what you want, but I noticed cdnjs does include it

Reply:
Oversight on my part, fixed.